src/NServiceBus.Core/Routing/MessageDrivenSubscriptions/MessageDrivenSubscriptions.cs

```diff
@@ -12,6 +16,12 @@ public class MessageDrivenSubscriptions : Feature
         internal MessageDrivenSubscriptions()
```

Comment:
maybe we should add a note to rename this class to `MessageDrivenPublishSubscribe` since it sets up both receiving and sending subscriptions and hooks into the publish pipeline.